src/domains.js

```diff
@@ -24,6 +24,10 @@ export function getPayPalDomain() : string {
     if (__ENV__ === ENV.SANDBOX) {
         return `${ getProtocol() }://www.sandbox.paypal.com`;
     }
+    
+    if (__ENV__ === ENV.SANDBOX_V2) {
```

Comment:
If you want to avoid modifying `ENV` could always just create a new temporary global flag like `__IS_SANDBOX_V2__`. Makes it easy to find/replace it later when we don't need it any more.